docs/_layouts/blog_post.html

```diff
@@ -23,31 +23,34 @@
 	{% include navbar.html %}
 		<!--main body-->
 		<div class="container">
+            <div class="row">
+                <div>
```

Comment:
Just a heads up, when using row in bootstrap, you have to have at least one nested col class (col-xs-12), even if it is just a 12 width column. Column adds a positive margin and row adds a negative one. Currently the title and body are misaligned b/c of this. I'm merging the branch and will just do a quick fix rather than kick this back but just wanted to flag it for you!